src/services/conversationsService.js

```diff
@@ -110,7 +137,7 @@ const searchPossibleConversations = async function(searchText, token, onlyUsers)
 	}
 
 	try {
-		return await axios.get(generateOcsUrl('core/autocomplete', 2) + `get`, {
+		return axios.get(generateOcsUrl('core/autocomplete', 2) + `get`, {
```

Comment:
😐 this changes the return from actual data to a promise. doesn't look intended. So better not hide this in a big feature PR?

Reply:
some leftover from a previous attempt, I'll remove